tests/Console/ConfigurationResolverTest.php

```diff
@@ -1038,6 +1000,37 @@ public function testResolveConfigFileOverridesDefault()
         $this->assertSame('xml', $resolver->getReporter()->getFormat());
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Expected "yes" or "no" for option "allow-risky", other values are deprecated and support will be removed in 3.0. Got "yes please", this implicitly set the option to "false".
+     */
+    public function testDeprecationOfPassingOtherThanNoOrYes()
+    {
+        $resolver = new ConfigurationResolver(
+            $this->config,
+            array('allow-risky' => 'yes please'),
+            ''
+        );
+
+        $this->assertFalse($resolver->getRiskyAllowed());
+    }
+
+    public function getResolveBooleanOptions()
```

Comment:
not important for sure.
on the other side, we follow that in basically all tests.